backend/src/models/workspace.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose')
+
+/**
+ * Partitions existing recruitment information into groups (aka "workspaces").
+ *
+ * name: name of the workspace. Functionally also a unique identifier.
+ * creator: name of the director who created the workspace
+ */
+
+const Workspace = new mongoose.Schema({
+  name: { type: String, required: true, unique: true },
```

Comment:
since this is user created, how will we make sure it unique on the frontend?

Reply:
🤔🤔🤔 will address this in frontend and API